test/e2e-helm/e2e_helm_suite_test.go

```diff
@@ -56,37 +48,12 @@ var _ = BeforeSuite(func(done Done) {
 	By("creating a new directory")
 	Expect(tc.Prepare()).To(Succeed())
 
-	By("checking the cluster type")
-	kubectx, err = tc.Kubectl.Command("config", "current-context")
+	By("getting the cluster Kind")
```

Comment:
same as above

Reply:
IMO the By make more sense in the tests to describe the step that will be called.